ui/src/app/pages/signed-in/component.ts

```diff
@@ -93,14 +94,25 @@ export class SignedInComponent implements OnInit, OnDestroy, AfterViewInit {
       this.minimizeChrome = minimizeChrome;
     }));
 
+    this.signOutIfLocalStorageInactivityElapsed();
```

Comment:
Does this get called when a sleeping machine is woken up? I think I got most of it but wondering how the check is enforced the moment the user returns to their machine.

Reply:
This line was actually just a refactoring of a check which was already here - no logic changed on this line. I didn't think hard about why it was here; I think this was to catch returning users.

It won't necessarily be enforced the moment they return to their machine, it will be enforced either:
1. When they next interact with the page: L175, or
2. when the interval check resets the timers (within <1m of machine wakeup): L163

Reply:
Got it. L175 was what I was looking for.